frontend/src/pages/user/UserAnswerPage.js

```diff
@@ -39,13 +41,48 @@ export default function UserAnswerPage() {
   const lastPage =
     questions && Array.isArray(questions) ? questions.length + 1 : 0;
 
+  const totalQuestion =
+    questions && Array.isArray(questions) ? questions.length : 0;
+  const passingScore = 75;
+
+  useEffect(() => {
+    setPassed(scorePercentage >= passingScore ? true : false);
+  }, [scorePercentage]);
+
+  useEffect(() => {
+    const calculate = async () => {
+      try {
+        const result = await calculateScore({
+          score,
+          totalQuestion,
+          answers,
+          correctAnswers,
```

Comment:
1. Why you are passing score when it is just `0`? remove it.
2. Remove the correctAnswers and TotalQuestions, just fetch it from BE.
3. You can just pass the question id to fetch those.